src/rocksdb.rs

```diff
@@ -1000,6 +1024,20 @@ impl Writable for DB {
     fn single_delete_cf(&self, cf: &CFHandle, key: &[u8]) -> Result<(), String> {
         self.single_delete_cf_opt(cf, key, &WriteOptions::new())
     }
+
+    fn delete_range(&self, begin_key: &[u8], end_key: &[u8]) -> Result<(), String> {
+        let batch = WriteBatch::new();
```

Comment:
It will not be a big performance decline and should not be an issue for TiKV. But jay still make his point clear. It's changed to use the `delete_range_cf` version for now. After https://github.com/facebook/rocksdb/pull/1960 is merged in next version of RocksDB, it should be replaced instead.